recipe/meta.yaml

```diff
@@ -1,28 +1,40 @@
-{% set name = "tokenizers" %}
 {% set version = "0.13.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: tokenizers
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3333d1cee5c8f47c96362ea0abc1f81c77c9b92c6c3d11cbf1d01985f0d5cf1d
+  url: https://github.com/huggingface/tokenizers/archive/refs/tags/python-v{{ version }}.tar.gz
```

Comment:
Is the goal to switch from PyPI to GitHub so that we can include and run the upstream test suite?

Reply:
Yes, because the tests are generally not part of the tarball (I don't specifically remember if I tested this for this feedstock, but I consider it cleaner than PyPI sources because there's one less layer where stuff can be spuriously changed).

Reply:
Yeah, makes sense.